### ISSUE TITLE
Add automation to unassign author from stale issues

### ISSUE THREAD

**Issue by nihal467** [coronasafe/care_fe#7231]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by thedevildude** [comment]
@nihal467 I found this workflow which can be used with our existing stale workflow to solve this issue. Do take a look at it. https://github.com/BoundfoxStudios/action-unassign-contributor-after-days-of-inactivity

Other approach I could think of was writing manual CLI commands to unassign issues.

**Comment by rithviknishad** [comment]
@thedevildude we already have the stale bot right? It supports output of the list of issues/prs marked as stale / closed due to inactivity (ref: https://github.com/actions/stale?tab=readme-ov-file#list-of-output-options).

Maybe we could use this output and do the necessary unassign based on the output from this itself?

**Comment by thedevildude** [comment]
@rithviknishad We can store the output from the stale bot and then use github-scripts actions to run a script that loops on the output and calls `github.issues.removeAssignees()`?

**Comment by rithviknishad** [comment]
yeah

**Comment by rithviknishad** [comment]
Add a comment too tagging the assignees why there were unassigned too.

**Comment by thedevildude** [comment]
@rithviknishad Got it, I'd like to work on if it's unblocked

**Comment by nihal467** [comment]
existing workflow looks good, upon re-evaluating, will be creating a new issue for better management, if required
